unleash-yggdrasil/src/strategy_parsing.rs

```diff
@@ -98,11 +98,18 @@ fn context_value(mut node: Pairs<Rule>) -> ContextResolver {
     }
 }
 
-// This is a special property, it's functionally identical to 'context_value'
-// and our rule here does nothing but call 'context_value' to resolve the value
-// this only gets a special place so that our syntax in the grammar is nice
-fn stickiness_param(mut node: Pairs<Rule>) -> ContextResolver {
-    context_value(node.next().unwrap().into_inner())
+pub(crate) fn coalesce_context_property(node: Pairs<Rule>) -> ContextResolver {
```

Comment:
For the unrusted, this function parses our context values into `(context) => Option<String>` functions and puts them in 'stickiness_resolvers' then moves them into a closure that executes them one by one, returning the results of the first one that returns a non null value